doc/visual-programming/source/widgets/data/pivot.md

```diff
@@ -0,0 +1,68 @@
+Pivot Table
+===========
+
+Reshape data table based on column values.
+
+**Inputs**
+
+- Data: input data set
+
+**Outputs**
+
+- Pivot Table: contingency matrix as set in the widget
+- Filtered Data: subset selected from the plot
+- Grouped Data: data table grouped by row values
+
+**Pivot Table** summarizes the data of a more extensive table into a table of statistics. The statistics can include sums, averages, counts, etc. The widget also allows selecting a subset from the plot and grouping by row values, which have to be a discrete variable. Data with only numeric variables cannot be displayed in the plot.
+
+![](images/Pivot-stamped.png)
+
+1. Discrete or numeric variable that will be used for row values. Numeric variables are considered as integers in this case. Variable values will appear as rows in the table.
+2. Discrete variable that will be used for column values. Variable values will appear as columns in the table.
+3. Values that will be used for aggregation. Aggregated values will appear as cells in the table.
+4. Aggregation methods:
+   - For any variable type:
+      - *Count*: number of instances that appear in the data
+      - *Count defined*: number of non-empty (not NaN) instances in the data.
+   - For numeric variables:
+      - *Sum*: sum of values.
+      - *Mean*: average of values.
+      - *Mode*: most frequent value of the subset.
+      - *Min*: smallest value.
+      - *Max*: highest value.
+      - *Median*: middle value.
+      - *Var*: variance of the subset.
+   - For discrete variables:
+      - *Majority*: most frequent value of the subset.
+5. Tick the box on the left to automatically output any changes. Alternatively, press *Apply* .
+
+Discrete variables
+------------------
+
+![](images/Pivot-discrete.png)
+
+Example of a pivot table with only discrete variables selected. We are using *heart-disease* data set for this example. We are using the values of *diameter narrowing* as row values, namely 0 and 1. Our columns are values of *gender*, namely female and male. We are using *thal* as values in our cells.
```

Comment:
`We are using the values of *diameter narrowing* as row values` -> `Rows correspond to values of *diameter narrowing* variable`.

You can skip `namely`.